mlflow/llama_index/__init__.py

```diff
@@ -363,6 +365,8 @@ def autolog(
         silent: If ``True``, suppress all event logs and warnings from MLflow during LlamaIndex
             autologging. If ``False``, show all events and warnings.
     """
+    from mlflow.llama_index.tracer import remove_llama_index_tracer, set_llama_index_tracer
```

Comment:
(for learning purposes) Why did you move the imports into the functions? Readability?

Reply:
Great question! This is indeed required for doc building. The python API doc is auto-generated from docstring, but in CI we don't install LlamaIndex (and many other ML libraries for other flavors), so if we do `import llama_index` at the top level it will fail with import error🙂